### PR TITLE
Use environment variables instead of console args.

### PR DESCRIPTION
**Pull request by hadrianvalentine**
So we can pass secrets though env vars.

### CODE REVIEW

**Review by itskingori — APPROVED** [review]

File: cmd/root.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ need to read from or write to the Vault instance.`,
 			// encrypt tokens with AWS KMS
 			fmt.Println("Encrypting root token...")
 			encryptedToken, errE := kmsClient.Encrypt(&kms.EncryptInput{
-				KeyId: aws.String(fullKeyID(os.Args[1], os.Args[2], region)),
+				KeyId: aws.String(fullKeyID(os.Getenv("AWS_ACCOUNT_NUMBER"), os.Getenv("AWS_KMS_KEY_ID"), region)),
```
Comment (zacblazic) [inline cmd/root.go:124]: Would be nice at some point to be able to support proper arguments in addition to environment variables.

Comment (zacblazic) [reply cmd/root.go:124]: There's a bunch of usages that would benefit:
* `VAULT_ADDR`
* `AWS_REGION`
* `AWS_ACCOUNT_NUMBER`
* `AWS_KMS_KEY_ID`
* `CHECK_INTERVAL`

Comment (zacblazic) [reply cmd/root.go:124]: It also seems that there's no handling around what happens when these environment variables are not set. 😅

Comment (itskingori) [reply cmd/root.go:124]: 💯%

**Review by zacblazic — APPROVED** [review]
Left some non-blocking comments.

File: cmd/root.go (hunk 1 above)
Comment (itskingori) [reply cmd/root.go:124]: Suggestion. We could have this in an issue and have it addressed later.

Comment (hadrianvalentine) [reply cmd/root.go:124]: Agreed. Will create an issue.

Comment (hadrianvalentine) [reply cmd/root.go:124]: Created them as separate issues so they can be resolved in separate PRs.

https://github.com/Intellection/vault-init/issues/6
https://github.com/Intellection/vault-init/issues/7

Please comment and edit as necessary.

**Pull request merged**